base/libgit2/tree.jl

```diff
@@ -16,21 +16,74 @@ function treewalk(f::Function, tree::GitTree, payload=Any[], post::Bool = false)
     return cbf_payload
 end
 
+"Get the filename of a tree entry."
 function filename(te::GitTreeEntry)
     str = ccall((:git_tree_entry_name, :libgit2), Cstring, (Ptr{Void},), te.ptr)
     str != C_NULL && return String(str)
     return nothing
 end
 
+"Get the UNIX file attributes of a tree entry."
 function filemode(te::GitTreeEntry)
     return ccall((:git_tree_entry_filemode, :libgit2), Cint, (Ptr{Void},), te.ptr)
 end
 
-
+"Convert a tree entry to the `GitAnyObject` it points to."
 function object(repo::GitRepo, te::GitTreeEntry)
     obj_ptr_ptr = Ref{Ptr{Void}}(C_NULL)
     @check ccall((:git_tree_entry_to_object, :libgit2), Cint,
                   (Ptr{Ptr{Void}}, Ptr{Void}, Ref{Void}),
                    obj_ptr_ptr, repo.ptr, te.ptr)
     return GitAnyObject(obj_ptr_ptr[])
 end
+
+"Get the id of the object pointed by the entry."
```

Comment:
These docstrings are so short and vague that they're not at all useful when trying to write code that uses these functions.